src/importexport/musicxml/internal/musicxml/musicxmlfonthandler.cpp

```diff
@@ -119,7 +119,8 @@ static int plainTextPlusSymbolsListSize(const std::list<TextFragment>& list)
  */
 
 bool MScoreTextToMXML::split(const std::list<TextFragment>& in, const int pos, const int len,
-                             std::list<TextFragment>& left, std::list<TextFragment>& mid, std::list<TextFragment>& right)
+                             std::list<TextFragment>& left, std::list<TextFragment>& mid, std::list<TextFragment>& right,
+                             const muse::modularity::ContextPtr& iocCtx)
```

Comment:
Now iocCtx is unused here (seems to become unnecessary in quite some cases in exportxml.cpp too)

(sorry for the big amount of time between my comments, I'm multitasking a bit)

Reply:
thank you, removed